win32ctypes/core/cffi/_kernel32.py

```diff
@@ -66,6 +74,18 @@ def _GetACP():
     return kernel32.GetACP()
 
 
+def _GetWindowsDirectory():
+    buffer = ffi.new('wchar_t[256]')
```

Comment:
if we can factor out a constant for `MAX_PATH` to be used for the cffi backend, that would be ideal in this case.